classes/Domain/Entity/TalkTag.php

```diff
@@ -21,7 +21,7 @@ public static function relations(\Spot\MapperInterface $mapper, \Spot\EntityInte
     {
         return [
             'talk' => $mapper->belongsTo($entity, \OpenCFP\Domain\Entity\Talk::class, 'talk_id'),
-            'tag'  => $mapper->belongsTo($entity, \OpenCFP\Domain\Entity\Tag::class, 'tag_id'),
```

Comment:
@chartjes 

Depending on what the preference is, we can also configure to align `=>` and `=` - what would you prefer?

Reply:
I think I like it all being lined up. 

I also think we should add some notes to the README for developers to let them know they should be running phpcs before every PR they submit.

Reply:
We have a note right here: https://github.com/opencfp/opencfp/blob/master/CONTRIBUTING.md#fixing-code-style-issues.